tests/integration_test.go

```diff
@@ -67,7 +67,10 @@ func generateBlocks(num uint64) {
 	}
 
 	fmt.Println("> Generating blocks for test setup...")
-	_ = d.GenerateBlocks(xmrmakerAddr.Address, num)
+	err = d.GenerateBlocks(xmrmakerAddr.Address, num)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
`GenerateBlocks` doesn't fail often like it used to in earlier versions of our code.